### DISCUSSION TITLE
Make it easier to access recently used distance spacing values

### DISCUSSION THREAD

**Discussion by cdwcgt** [ppy/osu#18474] [Ideas]
It may be necessary to use a variety of distance interval combinations when mapping
So, we can have a `bookmark list` for distance snap for easy switching
mapper can add different values to list and remove from list like image

![image](https://user-images.githubusercontent.com/34775378/170859060-b4c3a148-e209-460d-b2c4-b400796a6466.png)

the list can hide when not on cursor focus like the current design
![image](https://user-images.githubusercontent.com/34775378/170859141-6189fc04-c60f-4c74-a212-d41fcfe02e16.png)

Like `Ctrl+Alt+MWup(MWdown)`, we can use `Ctrl+Alt+(1\2\3)` to quickly switch to the list's settings

**Comment by bdach** [comment]
would sooner add a textbox there than do this.

**Comment by peppy** [comment]
I could see it being handy to include recently used values there for the current session (or for the current beatmap) in a flow (like a tag cloud). It would be completely automatic and not involve adding or removing entries.

**Reply by cdwcgt** [reply]
I'm glad you think about this, I look forward to the final design.
